<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-OIO4C93" failures="1" tests="2" name="Default test" time="0.026" errors="0" timestamp="2021-06-10T19:30:46 EDT">
  <testcase classname="collection.HashSet" name="test2" time="0.004">
    <failure type="java.lang.Error" message="Unresolved compilation problems: 
	The method add(int) is undefined for the type HashSet
	Syntax error, insert &amp;quot;VariableDeclarators&amp;quot; to complete LocalVariableDeclaration
	Syntax error, insert &amp;quot;;&amp;quot; to complete LocalVariableDeclarationStatement
	void is an invalid type for the variable $missing$
	The method test3() is undefined for the type HashSet
	Syntax error, insert &amp;quot;;&amp;quot; to complete Statement
	Duplicate local variable a
	The type MultidimensionalCounter.Iterator is not generic; it cannot be parameterized with arguments &amp;lt;String&amp;gt;
">
      <![CDATA[java.lang.Error: Unresolved compilation problems: 
The method add(int) is undefined for the type HashSet
Syntax error, insert "VariableDeclarators" to complete LocalVariableDeclaration
Syntax error, insert ";" to complete LocalVariableDeclarationStatement
void is an invalid type for the variable $missing$
The method test3() is undefined for the type HashSet
Syntax error, insert ";" to complete Statement
Duplicate local variable a
The type MultidimensionalCounter.Iterator is not generic; it cannot be parameterized with arguments <String>

at collection.HashSet.test2(HashSet.java:22)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- test2 -->
  <testcase classname="collection.HashSet" name="testMap" time="0.001"/>
</testsuite> <!-- Default test -->
